src/data/markdown/docs/30 xk6-browser/30 xk6-browser.md

```diff
@@ -37,69 +39,72 @@ If you're more adventurous or want to get the latest changes of the xk6-browser
 
 <InstallationInstructions extensionUrl="github.com/grafana/xk6-browser"/>
 
+## Launch
+
+The first step is to launch a [Browser](/javascript-api/xk6-browser/browser). After the browser starts, you can interact with it using the [browser-level APIs](#browser-level-apis). The only accepted browser type is currently `chromium`.
+
+| Method                         | Description                                                                                                                                                                                                         |
+|--------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| launch(browserName, [options]) | The first parameter is required, and the only accepted `browserName` is currently `chromium`. The `options` parameter is optional. You can see a list of all possible `options` in [the options heading](#options). |
+
+### Options
+
+| Method            | Description                                                                                                                                                                                              |
+|-------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| args              | Extra command line arguments to include when launching browser process. See [this link](https://peter.sh/experiments/chromium-command-line-switches/) for a list of Chromium arguments. Note that arguments should not start with `--` (see the [example](#Example)).  |
```

Comment:
I wish there was an official list somewhere instead of some random blog. I wasn't able to find another source though :-/